roles/openshift_metrics/templates/pvc.j2

```diff
@@ -25,3 +25,8 @@ spec:
   resources:
     requests:
       storage: {{size}}
+{% if storage_type is defined and storage_type %}
+  selector:
+    matchLabels:
+      storage-type: {{ storage_type }}
```

Comment:
could we make this more generic and allow the passing of a node selector? then one could provide "storage-type" as the selector key?